requirements_web.txt

```diff
@@ -1,7 +1,7 @@
 # Use this instead to test the latest fixes on 0.11.x branch before official release
 # git+https://github.com/biothings/biothings.api@0.11.x#egg=biothings[web_extra]
 
-biothings[web_extra]==0.11.1
+biothings[web_extra]==0.12.2
 
 # for sentry monitoring
 raven
```

Comment:
@everaldorodrigo I believe `raven` can be removed now, as `biothings` should install `sentry-sdk` package already.